web/src/reducers/songs/current.js

```diff
@@ -1,25 +1,34 @@
-import {INITIALIZE, SET_CURRENT_SONG} from '../../actions';
+import {INITIALIZE, SET_CURRENT_SONG, SET_SKIP_STATUS} from '../../actions';
 import {getUid} from './items';
 import {getSong} from '../songs';
 import {getUser} from '../users';
 
-function reduceSong(song) {
+function reduceSong(song, radioSkips) {
   if (!song) return null;
   return {
     song: getUid(song),
     dj: song.player.dj.id,
     startTime: song.player.startTime,
     offset: song.player.startTime - song.player.currentTime,
+    skips: radioSkips || 0,

```

Comment:
two things:

1. since we’re using a function arg for this, we can just default it to 0

2. if we rename it `skips`, this line can just be `skips,`

Reply:
i need to get better at visual code efficiency

Reply:
also :rabbit2: